backend/aml/urls.py

```diff
@@ -49,8 +49,8 @@
     urlpatterns = [path('server/', include(urlpatterns))]
 
 # Debug toolbar
-if settings.DEBUG:
+if settings.DEBUG and not settings.TESTING:
```

Comment:
NB: These changes were necessary as the debug toolbar was interfering with tests. So now, there's a distinction between debugging and testing. Side note: perhaps we might also just let the debug toolbar go, as it's not being really used by anyone?

Reply:
I never use it